plugins/postcss-custom-properties/test/basic.css

```
@@ -16,6 +16,7 @@ html {
 	--shadow: 0 6px 14px 0 color(var(--shadow-color) a(.15));
 	--font-family: "Open Sans", sans-serif;
 	color: var(--color);
+	--url: url("/my/path");
```

```suggestion
	--url: url("/my/path");
	color: var(--color);
```

Maybe we also need variations with `'` quotes and without any quotes?

Good list of examples : https://developer.mozilla.org/en-US/docs/Web/CSS/url()